server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -202,22 +202,16 @@
 }
 
 const uploadBlob = async (uri: string, blobData: git.ICreateBlobParams): Promise<git.ICreateBlobResponse> =>
-    new Promise<git.ICreateBlobResponse>((resolve, reject) => {
-        requestAPI(
-            {
-                body: blobData,
-                headers: {
-                    "Content-Type": "application/json",
-                },
-                json: true,
-                method: "POST",
-                uri,
-            },
-            (err, resp, body) => {
-                if (err) {
-                    reject(err);
-                } else {
-                    resolve(body as git.ICreateBlobResponse);
-                }
-            });
-    });
+  new Promise<git.ICreateBlobResponse>((resolve, reject) => {
+    axios.post(uri, blobData, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
```

Comment:
This might be angry because it sees the function param being plugged into the request directly.  Looks like there's only one callsite, maybe try inlining this there so it's more obvious that we don't just accept any URI but rather just one of our own construction.